app/src/ui/app.tsx

```diff
@@ -2293,7 +2304,8 @@ export class App extends React.Component<IAppProps, IAppState> {
   }
 
   private updateExistingLFSFilters = () => {
-    this.props.dispatcher.installGlobalLFSFilters(true)
+    const selectedRepository = this.props.appStore.getSelectedRepository()
+    this.props.dispatcher.installGlobalLFSFilters(true, selectedRepository?.codespace)
```

Comment:
Ditto about not understanding this one and not trusting this is doing remotely the right thing.